src/Kinepolis/Parser.php

```diff
@@ -7,7 +7,8 @@
 interface Parser
```

Comment:
Should this be a more specific name like movieParser?
I know it is in the Kinepolis namespace but this feels to generic.
If you don't feel that way - feel free to ignore this comment.

Reply:
I also think that a sub folder is needed with all the Parsers